assets/js/components/DashboardViewIndicator.stories.js

```diff
@@ -78,10 +83,14 @@ export default {
 			args.setupRegistry?.( registry );
 
 			return (
-				<WithTestRegistry registry={ registry }>
+				<WithTestRegistry
+					registry={ registry }
+					features={ [ 'ga4Reporting' ] }
```

Comment:
Hey @hussain-t, as mentioned on Slack, maybe we can use `enabledFeatures` here to avoid having to specify the feature flag twice. Please can you take a look?

https://github.com/google/site-kit-wp/blob/92e6409697d5d4edf7cb31833065f107e1d1241d/assets/js/components/notifications/SwitchGA4DashboardViewNotification.stories.js#L48

Reply:
I have updated the stories with `WithRegistrySetup`.